guides/common/modules/proc_configuring-the-base-operating-system-with-offline-repositories-in-rhel-8.adoc

```diff
@@ -0,0 +1,70 @@
+[id="configuring-the-base-operating-system-with-offline-repositories-in-rhel-8_{context}"]
+= Configuring the Base Operating System with Offline Repositories in RHEL 8
+
+Use this procedure to configure offline repositories for the {RHEL} 8 and {ProjectName} ISO images.
+
+.Procedure
+
+. Create a directory to serve as the mount point for the ISO file corresponding to the base operating system's version.
++
+[options="nowrap"]
+----
+# mkdir /media/rhel8-server
```

Comment:
there is no "rhel8 server", only "rhel8", so that's also what I would use in all directory names etc.